src/components/management/plugins/tabs/track/trackTab.jsx

```diff
@@ -671,13 +753,132 @@ class TrackTab extends React.Component {
   };
 
   pluginWindowClickHandler = (e) => {
-    e.preventDefault();
+   //  e.preventDefault();
     e.stopPropagation();
   };
   logClickHandler = (e) => {
     e.stopPropagation();
   };
 
+  prepareDropDownHtmlForPluginQueue = () => {
+    const list = this.state.pluginQueueList;
+    let options = [];
+    for (let i = 0; i < list.length; i++) {
+      options.push(
+        <option key={list[i].id} value={list[i].id}>
+          {'epadplugin_' + list[i].id}
```

Comment:
id is descriptive enough since the id is used for the docker container name and the user does not really need to know the container name. It is just a precaution to let the user container name so if something goes wrong we/user can debug it.